packages/plugin-ext/src/main/browser/webview/webview.ts

```diff
@@ -199,6 +199,15 @@ export class WebviewWidget extends BaseWidget implements StatefulWidget {
         this.toDisposeOnDetach.push(Disposable.create(() => this.forceHide()));
     }
 
+    protected onAfterAttach(msg: Message): void {
+        super.onAfterAttach(msg);
+        this.node.addEventListener('focus', () => {
```

Comment:
`this.addEventListener(this.node, ...` - otherwise a new listener added each time when a widget is attached, i.e. when it is moved between areas

Reply:
Thanks, fixed.